vmdk_plugin/utils/fs/fs_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fs_test
+
```

Comment:
The [moby](https://github.com/moby/moby/tree/master/volume) convention seems a little better. I could rename the package as just `fs`. The idea is that `fs.go`, `fs_linux.go` and `fs_windows.go` stay in a single directory and `Go` would [automatically pick](https://golang.org/pkg/go/build/) the correct set of files based on the platform for which the binary is being built.